engine/src/stack.h

```diff
@@ -300,6 +300,9 @@ class MCStack : public MCObject, public MCMixinObjectHandle<MCStack>
     
     // MW-2014-09-30: [[ ScriptOnlyStack ]] If true, the stack is a script-only-stack.
     bool m_is_script_only : 1;
+    
+    // BWM-2017-08-16: [[ Bug 17810 ]] Line endings for imported script-only-stack.
+    MCStringLineEndingStyle m_line_encoding_style;
```

Comment:
@peter-b 's comments are a little above my head there so I did some googling. I have found a number of suggestions not to use bitfields. Interestingly Mozilla's C++ portability guide recommends _only_ using them if they are all the same type and not to use them with enums. So we break both these rules here. https://developer.mozilla.org/en-US/docs/Mozilla/C++_Portability_Guide